pootle/apps/pootle_store/views.py

```diff
@@ -520,8 +523,29 @@ def submit(request, unit):
     form = form_class(request.POST, instance=unit, request=request)
 
     if form.is_valid():
+        suggestion = None
```

Comment:
as i understand this is related to accepting with alterations.

I dont totally follow what is going on here, but if you are going to land it can you add some tests